scipy/stats/tests/test_distributions.py

```diff
@@ -2789,6 +2789,53 @@ def test_minimum_data_zero_gh17801(self):
             _assert_less_or_close_loglike(dist, data, dist.nnlf)
 
 
+class TestPowerNorm:
+
+    # survival function references were computed with mpmath via
+    # from mpmath import mp
+    # x = mp.mpf(x)
+    # c = mp.mpf(x)
+    # float(mp.ncdf(-x)**c)
+
+    @pytest.mark.parametrize("x, c, ref",
+                             [(9, 1, 1.1285884059538405e-19),
+                              (20, 2, 7.582445786569958e-178),
+                              (100, 0.02, 3.330957891903866e-44),
+                              (200, 0.01, 1.3004759092324774e-87)])
+    def test_sf(self, x, c, ref):
+        assert_allclose(stats.powernorm.sf(x, c), ref, rtol=1e-13)
+
+    # inverse survival function references were computed with mpmath via
+    # from mpmath import mp
+    # def isf_mp(q, c):
+    #     q = mp.mpf(q)
+    #     c = mp.mpf(c)
+    #     arg = q**(mp.one / c)
+    #     return float(-mp.sqrt(2) * mp.erfinv(2*arg - 1))
+
+    @pytest.mark.parametrize("q, c, ref",
+                             [(1e-18, 2, 5.9978070150076865),
+                              (1e-50, 5, 6.361340902404057)])
```

Comment:
Ah, sorry, in my previous comment I said "We need a test that fails without the changes in this PR", but what I meant was "We need an *additional* test that fails without the changes in this PR".  Including the other tests that you had is fine, and probably preferable. Could you restore those?